.gitignore

```diff
@@ -192,3 +192,4 @@ wwwroot
 !src/Templates/**/content/**
 .template.config/
 /OrchardCore.sln.GhostDoc.xml
+/src/Templates/OrchardCore.ProjectTemplates/content
```

Comment:
These contents are used to build templates through the dotnet cli, so we update them periodically when related things are updated in our solution, so they need to not be ignored.

Reply:
Sorry @jtkech, that got added to my pull request somehow.   Kind of new at submitting these pull requests.   I will delete this file.

Reply:
no worries everybody do as they can.

But now the .gitignore is deleted, you should have just deleted the added line. But because now this file is deleted (but only in this branch) your next commit may have a bunch of files. You will have to clean your solution manually.

but no problem, if you have some troubles to do it, i will do it for you. But if so, the simplest way would be to restart from the dev and redo a new pr.

As a side note, maybe here you would have to create a kind of `IDbConnectionFactory` as i saw in the aspnet EF repo.

Reply:
I found IDbConnectionWrapper  in the Microsoft.ReportingServices.DataProcessing namespace.   It has a property called IDbConnection Connection.    I think this is essentially what @sebastienros  wants is a wrapper to call Task<IDbConnection> GetConnectionAsync() since it can't be called from services.AddScoped()  I will post what I have implemented it seems to work.